packages/config/config/devices/0x0371/zwa009.json

```diff
@@ -17,39 +17,37 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"associations": {
 		"1": {
 			"label": "Lifeline",
 			"maxNodes": 5,
 			"isLifeline": true
 		},
 		"2": {
-			"label": "High Temperature Notification",
+			"label": "High Temperature",
 			"maxNodes": 5
 		},
 		"3": {
-			"label": "Low Temperature Notification",
+			"label": "Low Temperature",
 			"maxNodes": 5
 		},
 		"4": {
-			"label": "High Humidity Notification",
+			"label": "High Humidity",
 			"maxNodes": 5
 		},
 		"5": {
-			"label": "Low Humidity Notification",
+			"label": "Low Humidity",
 			"maxNodes": 5
 		},
 		"6": {
-			"label": "Air Temperature",
-			"description": "A multilevel sensor report is sent to the nodes in this group",
+			"label": "Air Temperature (MultiLevel)",
 			"maxNodes": 5
 		}
```

Comment:
If I understand correctly, 2-5 send Notifications, 6 sends a Multilevel Sensor Report. Depending on what you want to do with the associations, having that info at a glance would be helpful I think

Reply:
On the other hand, zwave-js should probably just give applications a way to display which CCs which Association Group sends. The device is new enough so it should support that.

Reply:
But we should at least change `MultiLevel` to `Multilevel`